src/structs.rs

```diff
@@ -107,6 +107,63 @@ impl From<&AssetCode> for BaseField {
     }
 }
 
+/// Amount value type
+#[tagged_blob("AMOUNT_VALUE")]
```

Comment:
This causes it to serialize as TaggedBase64 in JSON. I think a more natural JSON serialization would just be a numeric string (which can be used without conversion as a "BigIntegerish" in ethers.js APIs). That would look something like
```rust
#[derive(Deserialize, Serialize)]
#[serde(from = "u128", into = "u128")]
```

Reply:
Sounds good, done in a8026b4d31ea64378a50d0548dd63587da738163